ansible/dualtor/y_cable_simulator_client.j2

```diff
@@ -19,6 +24,115 @@ helper_logger = logger.Logger(SYSLOG_IDENTIFIER)
 UPPER_TOR = "upper_tor"
 LOWER_TOR = "lower_tor"
 
+PLATFORM_JSON = 'platform.json'
+PORT_CONFIG_INI = 'port_config.ini'
+
+# A dict for mapping physical port to host interface
+g_physical_to_host_port_mapping = {}
+g_physical_to_logical_port_mapping = {}
+
+def _physical_port_to_host_port(physical_port):
+    """
+    Convert physical port to host interface index.
+    The physical port index got from spf is different with host interface index when interface is splited.
+    However, the mux bridges on vm_host are named according to host interface index (vm_topology.py).
+    So a convert is needed.
+
+    @arg physical_port: The physical port index got from sfputil
+    @returns: The host interface index (0-based)
+    """
+    global g_physical_to_host_port_mappingg
+    global g_physical_to_logical_port_mapping
+
+    if not g_physical_to_host_port_mapping or not g_physical_to_logical_port_mapping:
+        _load_port_info()
+    return g_physical_to_host_port_mapping[physical_port]
```

Comment:
This is unlikely to happen, bu t if _load_port_info() failed, you probably should return the old (physical_port - 1) as it still works on some platforms?

Reply:
Updated. Thanks!